### DISCUSSION TITLE
sincronización fallida QFIELD

### DISCUSSION THREAD

**Discussion by Lymantria23** [opengisch/QField#2600] [Q&A]
Hola, soy @Lymantria23 estoy empezando con el QFIELD y tengo problemas con la sincronización. Preparé el proyecto base y lo envié a los móviles que hacen un censo de plagas, ellos rellenan los campos solicitado y me lo envía de vuelta. En mi ordenador sincronicé una ciudad "A" y fue todo correcto, el problema es con el resto de ciudades, en mi ordenador cuando le indico sincronizar "B" me aparecen los datos de "A", aunque los que me envía el agente son de "B" ( lo he comprobado) por lo que entiendo me repite la primera sincronización que hice. He probado a ver si la dirección de sincronización es correcta y lo tengo dirigido a la carpeta correspondiente. He probado con otro ordenador para ver si daba el mismo fallo y en ese ordenador que nunca se ha hecho una sincronización, dice que está correcta pero me muestra la pantalla en blanco y no sale ninguna de las capas. Por favor necesito ayuda urgente, tengo los datos de 25 personas por sincronizar y no sale ninguna. Mi QFIELD es 1.10.0 y el QGIS es 3.4.8. Gracias

**Comment by suricactus** [comment]
Please translate your message in English. Thanks.
